.circleci/config.yml

```diff
@@ -118,6 +118,19 @@ workflows:
             tags:
               only: /^v.*/
 
+      - architect/push-to-app-collection:
+          context: architect
+          name: push-to-capz-app-collection
+          app_name: "falco-app"
```

Comment:
I think you need to update your local branch. The app has been renamed to just `falco` -- check main: https://github.com/giantswarm/falco-app/blob/main/.circleci/config.yml#L99

Reply:
Oooops! You are right! Fixed.